front/app/containers/Admin/users/routes.tsx

```diff
@@ -11,7 +12,8 @@ import { AdminRoute } from '../routes';
 
 export enum usersRoutes {
   users = 'users',
-  adminsManagers = 'admins-managers',
+  admins = 'admins',
+  managers = 'managers',
   groupId = `:groupId`,
```

Comment:
@IvaKop We're using two terms here - "Moderators" and "Managers".. do you think there's any way to just use one or the other? :thinking:

Reply:
Very good point! We prefer manager in the copy but the code refers to moderators so it can be a bit confusing. I changed the route to "moderators" in the code to make it more consistent 👍